src/NSwag.AspNet.Owin/SwaggerUi3/index.html

```diff
@@ -45,6 +45,18 @@
   }
   var urls = {Urls};
 
+  const disableTryItOutPlugin = function() {
+    return {
+      statePlugins: {
+      spec: {
+        wrapSelectors: {
+            allowTryItOutFor: () => () => !{DisableTryItOut}
```

Comment:
Please use ES3 javascript to support all browsers (ie old school functions instead of lambda)

Reply:
Yep! Will do